packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -40,17 +40,14 @@ class AndroidStudio implements Comparable<AndroidStudio> {
     _init(version: version);
   }
 
-  factory AndroidStudio.fromMacOSBundle(String bundlePath) {
-    String studioPath = globals.fs.path.join(bundlePath, 'Contents');
-    String plistFile = globals.fs.path.join(studioPath, 'Info.plist');
-    Map<String, dynamic> plistValues = globals.plistParser.parseFile(plistFile);
+  static AndroidStudio? fromMacOSBundle(String bundlePath) {
+    final String studioPath = globals.fs.path.join(bundlePath, 'Contents');
+    final String plistFile = globals.fs.path.join(studioPath, 'Info.plist');
+    final Map<String, dynamic> plistValues = globals.plistParser.parseFile(plistFile);
     // As AndroidStudio managed by JetBrainsToolbox could have a wrapper pointing to the real Android Studio.
     // Check if we've found a JetBrainsToolbox wrapper and deal with it properly.
```

Comment:
Update the comment too.  Maybe "If we've found a JetBrainsToolbox wrapper, ignore it."

Reply:
done